src/main.rs

```diff
@@ -1,27 +1,29 @@
-use std::error::Error;
 use std::io;
-use std::process;
 
+use anyhow::Context;
 use csv::StringRecord;
 
-fn main() {
+fn main() -> anyhow::Result<()> {
```

Comment:
We can tell rust to return an error instead of manually handling error's and exits.

If we do not want to do this we should at least be [DRY](https://www.digitalocean.com/community/tutorials/what-is-dry-development) and make a method that we use for exiting with a non-zero code with a message.